src/components/ConnectWalletButton.tsx

```diff
@@ -22,30 +23,42 @@ export function ConnectWalletButton() {
               <Flex>
                 <Button
                   variant="ghost"
-                  onClick={openChainModal}
+                  _hover={{ bgColor: 'blue.1000' }}
+                  _focus={{ bgColor: 'blue.1000' }}
+                  cursor="auto"
+                  mx={2}
+                  bg="blue.1000"
                 >
-                  <Text color="brand.500">{chain.name}</Text>
-                  <ChevronDownIcon ml={1} />
+                  <Image
+                    mr={2}
+                    height={'20px'}
+                    src="sarco-token-icon.png"
+                  />
+                  <Text>{formattedBalance}</Text>
```

Comment:
I had to use a button here because `Flex` wasn't expanding to  fit the sarco balance text -- it would always render on 2 separate lines. Solutions welcome. Struggled with this one.